templates/tag_summary_template.html

```diff
@@ -1,222 +1,234 @@
+{% else %}
+<div class="lj-scribe-tag" id="{{ orig_str.replace('@', '') }}">
+   {% if tags[0][0].title != "" %}
+   <h5 class="tag-title">{{ tags[0][0].title }}</h5>
+   {% endif %}
+   <table class="tag-summary">
+      <thead>
+         <tr class="emphasis-row">
+            <td width="80%">Name</td>
+            <td>Start Address</td>
+            <td>Type</td>
+            <td>Access</td>
+         </tr>
+      </thead>
+      <tbody>
+         {% for (orig_subtag, subtag) in tags %}
+         <tr class="class-summary expand" onclick="hideDetails('{{subtag.resolved[0].address}}','{{subtag.resolved[0].address}}IMG')">
+            <td >
+               <p class="shortdes" >
+                  <a onclick="hideDetails('{{subtag.resolved[0].address}}','{{subtag.resolved[0].address}}IMG')">
+                  <img src="https://ljsimpleregisterlookup.herokuapp.com/static/images/details_open.png" id="{{subtag.resolved[0].address}}IMG" onclick="hideDetails('{{subtag.resolved[0].address}}','{{subtag.resolved[0].address}}IMG')"></a>
+                  {{ get_full_name(orig_subtag) }} &nbsp;&nbsp; &nbsp;&nbsp; &nbsp;&nbsp; &nbsp;&nbsp; 
+                  <span id="{{subtag.resolved[0].address}}SPN"> {{ subtag.resolved[0].description | safe }}</span>
+               </p>
+            </td>
+            <td>{{subtag.resolved[0].address}}</td>
+            <td>{{subtag.unresolved.type}}</td>
+            <td>
+               {% if subtag.resolved[0].read %}
+               {% if subtag.resolved[0].write %}
+               R/W
+               {% else %}
+               R
+               {% endif %}
+               {% elif subtag.resolved[0].write %}
+               W
+               {% endif %}
+            </td>
+         </tr>
+         <tr class="expandbody">
+            <td id="{{subtag.resolved[0].address}}" style="display:none;" class="printexpand">
+               <div>
+                  {{ registerBody(orig_subtag, subtag) }}
+                  </br>
+               </div>
+            </td>
+         </tr>
+         {% endfor %}
+      </tbody>
+   </table>
 </div>
 <script>
-    function hideDetails(z) {
-    var id = z
-    var x = document.getElementById(id);
-    if (x.style.display === "none") {
-        x.style.display = "block";
-    } else {
-        x.style.display = "none";
-    }
-    }
+   function hideDetails(id, imgid) {
+   var x = document.getElementById(id);
+   var buttonimg = document.getElementById(imgid);
+   var span = id + "SPN";
+   span = document.getElementById(span);
+   if (x.style.display === "none") {
+       x.style.display = "block";
+       span.style.display = "none";
+       buttonimg.src = "http://ljsimpleregisterlookup.herokuapp.com/static/images/details_close.png"
+   } else {
+       x.style.display = "none";
+       span.style.display = "inline";
+       buttonimg.src = "https://ljsimpleregisterlookup.herokuapp.com/static/images/details_open.png"
+   
+   }
+   }
 </script>
- {% endif %}
+{% endif %}
```

Comment:
Since the indentation has been changed here, it's hard to see what actually changed in the code. I'm just going to call it fine. This means I'm not reviewing this section of code. Please take care in the future to not mess with existing indentation so that diffs are minimal.

tldr; I'm assuming you code is flawless!

Reply:
I re indented the whole file, it was pretty bad before I started, I have a version that only has my changes. Would you like me to use this one?